Sources/MySQLKit/MySQLDatabase.swift

```diff
@@ -1,5 +1,7 @@
 @_exported import struct Foundation.URL
+@_exported import struct Foundation.Data
 @_exported import struct NIOSSL.TLSConfiguration
+import CryptoKit
```

Comment:
To follow the importing pattern with the rest of the Vapor repos, this should probably something like this:
```suggestion
import enum CryptoKit.Insecure
```